controllers/halayerset_controller_test.go

```diff
@@ -277,13 +325,14 @@ func verifyHAPodExist() bool {
 func verifyCRIsCreated() bool {
 	return Eventually(
 		func() bool {
-			_, err := getHALayerCR()
-			return err == nil
+			hals, err := getHALayerCR()
+			return err == nil && hals.Status.PrevFenceAgentsSpec != nil
 		}, time.Second*10, time.Millisecond*10).Should(BeTrue())
```

Comment:
Just for easier reading...
Maybe replace the number `10` or the whole word with a Macro, I saw it is repeated plenty of times in your tests.
For example, see https://github.com/razo7/githubissues-operator/blob/main/controllers/githubissue_controller_test.go#L90

Reply:
In this particular case I disagree.
I don't think using a constant here will improve readability IMO using  `time.Second*10`  give a very clear sense on how long the delay is contrary to `Interval`.

Another very common  reason to use a constant could be to prevent the need to update the value in multiple places.
However  I don't think it applies here since each timeout is relevant for a different use-case and I wouldn't want a single change to effect all of them.